src/location/__tests__/components/LoginMapView.test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import LoginMapView from '../../components/LoginMapView';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as ReactReduxHooks from '../../../hooks/reduxHooks';
+import { Provider } from 'react-redux';
+import { mapUtility } from '../../../utils/mapUtility';
+
+
+jest.mock('mapbox-gl', () => {
+    const mockMap = jest.fn(() => ({
+        on: jest.fn(),
+        off: jest.fn(),
+        getSource: jest.fn(),
+        getLayer: jest.fn(),
+        addControl: jest.fn(),
+        addSource: jest.fn(),
+        addLayer: jest.fn(),
+        addImage: jest.fn(),
+        loadImage: jest.fn(),
+        hasImage: jest.fn(),
+        dragPan: {
+            enable: jest.fn(),
+            disable: jest.fn()
+        },
+        scrollZoom: {
+            enable: jest.fn(),
+            disable: jest.fn()
+        },
+        doubleClickZoom: {
+            enable: jest.fn(),
+            disable: jest.fn()
+        },
+        touchZoomRotate: {
+            enable: jest.fn(),
+            disable: jest.fn()
+        },
+        setStyle: jest.fn(),
+        queryRenderedFeatures: jest.fn(),
+        getCenter: jest.fn(),
+        flyTo: jest.fn(),
+        isStyleLoaded: jest.fn(),
+        getBounds: jest.fn(),
```

Comment:
To avoid repetition, you could move this out to a separate folder called mocks I guess, please check documentation.